ccsdspy/interface.py

```diff
@@ -121,3 +144,34 @@ def load(self, file, include_primary_header=False):
 
         field_arrays = _decode_fixed_length(file_bytes, self._fields)
         return field_arrays
+
+
+def _get_fields_csv_file(csv_file):
+    """Parse a simple comma-delimiated file that define a packet. Should not include the CCSDS header.
+    Accepts 3 column files (name, data_type, bit_length) or 4 column files (name, data_type, bit_length, bit_offset).
+    A header line is required.
+
+    Parameters
+    ----------
+    csv_file: str
+        Path to file on the local file system, or file-like object
+
+    Returns
+    -------
+    fields: list
+        A list of `PacketField` objects.
+    """
+
+    with open(csv_file, "r") as fp:
```

Comment:
This function would be better implemented with the `csv` module `DictReader` class, so that things like quoted strings are supported and the order of columns doesn't matter.
https://docs.python.org/3/library/csv.html#csv.DictReader

Reply:
No problem. Updated to use `csv` module.